PyApps/src/Plugins/realvsimag_qt5.py

```diff
@@ -764,28 +761,28 @@ def onMousePressed(self, e):
 # Now create text marker giving source of point that was clicked
         self.marker = self.plot.insertMarker()
         try:
-          ylb = self.plot.axisScale(Qwt.QwtPlot.yLeft).lowerBound()
-          xlb = self.plot.axisScale(Qwt.QwtPlot.xBottom).lowerBound()
+          ylb = self.plot.axisScale(QwtPlot.yLeft).lowerBound()
+          xlb = self.plot.axisScale(QwtPlot.xBottom).lowerBound()
         except:
-          ylb = self.plot.axisScale(Qwt.QwtPlot.yLeft).lowerBound()
-          xlb = self.plot.axisScale(Qwt.QwtPlot.xBottom).lowerBound()
+          ylb = self.plot.axisScale(QwtPlot.yLeft).lowerBound()
+          xlb = self.plot.axisScale(QwtPlot.xBottom).lowerBound()
```

Comment:
Don't follow the logic here? You are invoking the same code on an exception. Also possibly narrow the catch clause to only execute on intended exceptions?